product_docs/docs/net_connector/8.0.5.1/release_notes/index.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: "Release notes"
+navigation:
+- 8.0.5.1_rel_notes
+- 8.0.2.1_rel_notes
+- 7.0.6.2_rel_notes
+- 7.0.6.1_rel_notes
+- 7.0.4.1_rel_notes
+- 6.0.2.1_rel_notes
+- 5.0.7.1_rel_notes
+- 4.1.6.1_rel_notes
+- 4.1.5.1_rel_notes
+- 4.1.3.1_rel_notes
+- 4.0.10.2_rel_notes
+- 4.0.10.1_rel_notes
+- 4.0.6.1_rel_notes
+---
```

Comment:
Need a redirect from `01_release_notes/` (the former name of the release notes path for net_connector) here.
```suggestion
redirects:
- ../01_release_notes/
---
```